database/oracle/README.md

```diff
@@ -0,0 +1,87 @@
+# oracle
+The golang oracle driver [godror](https://github.com/godror/godror) does not natively support executing multiple statements in a single query. 
+Here are the strategies for splitting the migration text into separately-executed statements if multi statements detecting isn't been disable explicitly via `x-disable-multi-statements`:
+1. If there is no PL/SQL statement in a migration file, the `semicolons` will be the separator
+1. If there is any PL/SQL statement in a migration file, the separator will be `---` in a single line or specified by `x-plsql-line-separator`, 
+   And in this case the multiple statements cannot be used when a statement in the migration contains a string with the given `line separator`.
+
+`oracle://user:password@host:port/sid?query`
+
+| URL Query  | WithInstance Config | Description |
+|------------|---------------------|-------------|
+| `x-migrations-table` | `MigrationsTable` | Name of the migrations table in UPPER case |
+| `x-disable-multi-statements` | `DisableMultiStatements` | Indicate if disable multi statements detection automatically |
+| `x-plsql-line-separator` | `PLSQLStatementSeparator` | a single line which use as the token to spilt multiple statements in single migration file (See note above), default `---` |
+
+## Runtime dependency
+
+Although an Oracle client is NOT required for compiling, it is at run time. One can download it from https://www.oracle.com/database/technologies/instant-client/downloads.html
+
+## Supported & tested version
+- 12c-ee
```

Comment:
Add language about how this driver is not supported by the migrate project/organization.

Reply:
resolved